### ISSUE TITLE
Bad site - fishing

### ISSUE THREAD

**Issue by coinzzzpro** [MetaMask/eth-phishing-detect#1921]
Hello! We do not understand why our site https://coinzzz.pro/ was included in the list of phishing. We don't have anything like that, just a text site. We even deleted the links to Airdrops just in case. Please help!

**Comment by 409H** [comment]
You were hosting a fake Aragon giveaway. See commit https://github.com/MetaMask/eth-phishing-detect/commit/ed212d4719ada0b774f96b84de49c73b7af51b53

```
coinzzz.pro
Hosting a fake Aragon giveaway (https://t.me/Aragonofficial & https://t.me/Aragonannoucement)
https://urlscan.io/result/c671b699-419f-49c1-babe-3ae336bc9ea4/
https://urlscan.io/result/e81cdbc9-812f-4937-88d3-7392b4a4cbb5/
```

**Comment by coinzzzpro** [comment]
Thank you!!!

I deleted this Airdrop! Like all Airdrops - among them a lot of scams. Please tell me how we now disappear from the list of phishing sites?

**Comment by 409H** [comment]
> I deleted this Airdrop!

Do you not vet the information before you publish?

**Comment by coinzzzpro** [comment]
Yуы, sure we are. But we didn't work with drops before and my employee added it. We won't do that again. Forgive me! We are not scammers or hackers.

**Comment by coinzzzpro** [comment]
I found the source where we got that drop: 
![zzz2018-06-19 23 26 13](https://user-images.githubusercontent.com/40394013/41622477-5bba4e2a-7418-11e8-8280-3e43a0bee90d.jpg)

**Comment by coinzzzpro** [comment]
The fate of the site depends on you. Please have mercy!

**Comment by coinzzzpro** [comment]
Our site has already been excluded from its databases: Kaspersky and Phishtank. Please exclude us from your list!

**Comment by coinzzzpro** [comment]
Please answer me.

**Comment by coinzzzpro** [comment]
You blocked the website, and the links from which you locked open. 
![screenshot at 26 14-36-05](https://user-images.githubusercontent.com/40394013/41908994-7473c1ea-794e-11e8-9834-276b61ab2ea1.png)

**Comment by coinzzzpro** [comment]
Please answer me.

**Comment by coinzzzpro** [comment]
Why are you ignoring me? You have a personal grudge against me? Please pay attention.

**Comment by coinzzzpro** [comment]
We published a press release with an apology to our readers. https://coinzzz.pro/nash-sayt-popal-v-spisok-fishingovyih-saytov-plagina-metamask/

**Comment by 409H** [comment]
What is your strategy to combat posting more fake airdrops/phishing links that could harm your readers?

**Comment by coinzzzpro** [comment]
Now we will publish only data from official project sites (Thank you for your software, we will check through it too) and and in parallel to check the site for the presence in the database phishtank.com.
But the airdrops we are not going to publish.

**Comment by coinzzzpro** [comment]
Please exclude my site from the list, I lose my work and visitors. The project dies.

**Comment by kumavis** [comment]
removed
https://github.com/MetaMask/eth-phishing-detect/commit/5802be01c06dddff18d8077dd6f3cecf80dde9f5

**Comment by coinzzzpro** [comment]
Thank you!
